Gordon360/ApiControllers/ProfilesController.cs

```diff
@@ -369,10 +369,14 @@ public IHttpActionResult GetEmergencyContact(string username)
         /// <summary>Gets the mailbox information of currently logged in user</summary>
         /// <returns></returns>
         [HttpGet]
-        [Route("mail-info/{boxNumber}")]
-        public IHttpActionResult GetMailInfo(string boxNumber)
+        [Route("mailbox-combination")]
```

Comment:
Make sure to update this route on the README file as well.

Reply:
There was off-line discussion of whether this info should be included in the self-profile, instead of having a separate API endpoint.  The argument for the code as-is (new API endpoint) is simplicity: the security model is very clear, and the code is here and works.  The argument for instead returning the combination with the person's own profile is performance (fewer round trips) and possibly simplicity of the my-profile code (having all the info from one call).  

Since we aren't aware of any performance problems, and since this code is clear, and since the practicum is almost over, let's commit this rather than trying something different.